README.md

```diff
@@ -60,9 +47,9 @@
 ### Key Features <a name="key-features"></a>
 
 <!-- > Describe between 1-3 key features of the application. -->
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[adding index.html file]**
+- **[adding styles to html]**
+- **[adding submit button form]**
 
```

Comment:
- [OPTIONAL ] You can take the square brackets away when you are done editing this section. 🤪

Reply:
- [OPTIONAL ] Please resolve this conflict so that all your files are up to date

Reply:
Noted, i will fix this by remove the square brackets